src/util/vscodePrompts.ts

```diff
@@ -146,7 +147,10 @@ export async function askForMPSpecifications(
   return undefined;
 }
 
-export async function askForFolder(customOptions: OpenDialogOptions): Promise<Uri | undefined> {
+export async function askForFolder(
```

Comment:
I think it might be better to leave `askForFolder` as is and create a new function called `askForTargetFolder` that wraps `askForFolder` and adds the extra logic. It will keep the `askForFolder` function simpler as it only does 1 thing.